oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -51,52 +51,96 @@ Reports come into security@openedx.org
+
+.. _GitHub repository security advisory: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/creating-a-repository-security-advisory
+
+Details of security vulnerabilities are private until a fix is released
+-----------------------------------------------------------------------
+Members of the Open edX community with access to the details of a disclosure are asked to keep those details private until a fix is publically released for the vulnerability.
+
+Maintainers should use GitHub's `temporary private fork`_ feature within the GitHub repository security advisory created by the security working group to keep the implementation details of a fix for a vulnerability private.
+
+.. _temporary private fork: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability
+
+Operators are ultimately responsible for the security of their Open edX instance
+--------------------------------------------------------------------------------
+The security working group will put forth good faith effort to responsibly forward disclosures related to the operation of a specific Open edX instance to the relevant operator.
+
+Operators wishing to register a desired primary point of contact for security-related matters should email security@openedx.org with the name, domain, contact information for an individual point of contact for security, and an email address to which security reports should be forwarded.
+
+Operators are encouraged to forward or to invite reporters to forward to security@openedx.org any security issue they receive that may be relevant to the Open edX project.
 
 .. _security-releases:
 
-Security Releases
------------------
+Maintainers will work together with the on-call security working group member to release the security fix
+---------------------------------------------------------------------------------------------------------
 The current process for releasing security fixes involves sending a disclosure and security patch to members of the
 open source security email list and waiting two days before making the patch public.  Instead of this process, we
 propose the following security release process:
 
-#. A notice will be sent to everyone to announce an upcoming security patch, specifying the date and time at which
-   the patch will become public
+#. The maintainer will email security@openedx.org notifying that a fix is ready to be merged and the date and time of when they would like to merge it. The merge time should be at least 72 hours after the security patch is announced.
+
+#. The on-call security working group member will create an announcement post in the `Security Announcement Section`_ on https://discuss.openedx.org. It should specify the affected repository, the date and time at which the patch will become public, and the severity of the vulnerability it fixes.
```

Comment:
Gut reaction: This feels like something we could just ask the maintainer to do if we provide them a doc to follow, but I'm fine with it being a SWG responsibility if you feel strongly about it.

Reply:
Not at all, I prefer less work honestly. Amended.